packages/salesforcedx-vscode-core/src/commands/util/libraryCommandlet.ts

```diff
@@ -48,6 +53,7 @@ export abstract class LibraryCommandletExecutor<T>
       throw new Error(nls.localize('error_no_default_username'));
     }
     const conn = await OrgAuthInfo.getConnection(usernameOrAlias);
+    // @ts-ignore private logger mismatch
```

Comment:
Isn't the private logger problem throwing issues at runtime ?

Reply:
It doesn't appear to be